docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -109,15 +109,15 @@ implementation(group: 'com.microsoft.sqlserver', name: 'mssql-jdbc', version: '1
 .environment.properties
 [source,properties]
 ----
-db.connection.vividus.url=jdbc:sqlserver://mssql.db.server:1433\mssql_instance;databaseName=my_database
+db.connection.vividus.url=jdbc:sqlserver://;serverName=mssql.db.server;databaseName=my_database;encrypt=true;trustServerCertificate=true
 db.connection.vividus.username=vividus
 db.connection.vividus.password=pa$$w0rd
 ----
 
 .https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url[URL template]
 [source,properties]
 ----
-jdbc:sqlserver://serverName\instanceName:portNumber;parameter1=value1;parameter2=value2
+jdbc:sqlserver://;servername=server_name;encrypt=true;integratedSecurity=true;parameter1=value1;parameter2=value2
```

Comment:
Why are `encrypt` and `integratedSecurity` parameters needed?

Reply:
for the secure connection

Reply:
from the MS documentation
![image](https://user-images.githubusercontent.com/30229905/218055304-997956ee-e255-4fba-83a7-e67e58495557.png)

Reply:
`integratedSecurity` is usually for Windows or require another parameters